docs/twig-components/method/floatval.md

```diff
@@ -1,6 +1,65 @@
 # floatval
 
-`floatval` is a Twig filter to ...
+`floatval` is a Twig filter to convert a variable to a floatval. Converting a variable to a specific type is not
+something Twig encourages and it probably should be avoided, if possible. Yet there are situations where you just want
+to convert something to an integer or string so you can be sure a comparison is type safe or that there is no unexpected
+behavior because one value has the wrong type.
 
+```twig
+{% if '5.7'|floatval === 5.7 %}
+Convert '5.7' to a float - this if block is being executed
+{% endif %}
+```
 
-Source: Squirrelphp
\ No newline at end of file
+These filters mainly behave like the ones in PHP (and use the corresponding PHP functions internally), but there is some
+additional behavior to detect or avoid likely errors:
+
+- only scalar values, null and objects with a __toString method are allowed, so if you use any of these filters with an
+  array or an object that cannot be cast to a string it will throw an exception
+- null will return 0 for intval, '' for strval, false for boolval and 0.0 for floatval (just like in PHP)
+- objects with a __toString method will be converted to a string first (using the __toString method), and only after
+  that intval, boolval and floatval will be used
+
+### Installation
+
+```shell
+composer require squirrelphp/twig-php-syntax
+```
+
+### Configuration
+
+Add PhpSyntaxExtension to Twig:
+
+```twig
+$twig = new \Twig\Environment($loader);
+$twig->addExtension(new \Squirrel\TwigPhpSyntax\PhpSyntaxExtension());
+```
+
+You can also have a look at the extension definition and create your own extension class to only include some of the
+features, if you do not like all of them.
+
+Symfony integration If you use `autoconfigure` (which is the default) you just need to load the PhpSyntaxExtension class
```

Comment:
The start of this paragraph (Symfony integration) sounds like it is not related to what comes next (If you use `autoconfigure` ...)

Reply:
Or maybe a colon (:) is missing after `Symfony integration` ?